database/Restaurant.js

```diff
@@ -0,0 +1,86 @@
+const mongoose = require('mongoose');
+const db = require('./index.js');
+const faker = require('faker');
+mongoose.Promise = global.Promise;
+
+
+const restaurantSchema = new mongoose.Schema({
+  name: String,
+  users: [{
+    username: String,
+    profilePic: String,
+    city: String
+  }],
+  reviews: [{
+    review: String,
+    createdAt: Date,
+    ratings: [{
+      overall: Number,
+      food: Number,
+      ambiance: Number,
+      service: Number,
+      value: Number
+    }]
+  }]
+});
+
+
+class User {
+  constructor() {
+    this.username = faker.internet.userName(),
+    this.profilePic = faker.internet.avatar(),
+    this.city = faker.address.city()
+  }
+};
+
+const makeUser = () => {
+  let n = Math.floor(Math.random() * 40 + 1);
```

Comment:
@kevinphm @jfeng3124 We can come up with a more semantic variable name than "n."
"const" is a more appropriate variable type, since this variable will never be reassigned. AirBnB's style guide will point this out incessantly. Careful about "magical numbers" like 40 + 1